README.md

```diff
@@ -1,13 +1,13 @@
-# createRulesetModelDescription
+# createRulesetProjectDescription
```

Comment:
If it's helpful, this title could be a proper title rather than the repo name.

Reply:
Good idea.